src/frontend/src/pages/core/CoreIndex.tsx

```diff
@@ -8,8 +8,8 @@ import { PageDetail } from '../../components/nav/PageDetail';
 import { PanelGroup } from '../../components/panels/PanelGroup';
 import { useUserState } from '../../states/UserState';
 import { ContactTable } from '../../tables/company/ContactTable';
-import { UserTable } from '../../tables/core/UserTable';
 import { GroupTable } from '../../tables/settings/GroupTable';
+import { UserTable } from '../../tables/settings/UserTable';
```

Comment:
These tables had very different table actions and fields - was that considered?

Reply:
Apologies, I thought they were effectively the same thing. I'll look at this again.